src/components/OrgChart/OrgChart.md

```diff
@@ -0,0 +1,23 @@
+# Org Chart
+Displays multiple Persona components in groups with headers.
```

Comment:
I would consider:
"Displays multiple Persona components in groups with headers in order to show the hierarchy of an organization or team."